### ISSUE TITLE
The given Hoogle database is not available.

### ISSUE THREAD

**Issue by gitfoxi** [fpco/stackage-server#104]
I get this message sometimes after querying the Hoogle beta.

It looks like this might be a capacity issue for the server since it's responding this way only sometimes.

**Comment by ketzacoatl** [comment]
@gitfoxi, can you confirm the URLs you see as working or failing? Or do you see the same URLs as sometimes working and sometimes failing?

I consistently see https://beta.stackage.org/lts-2.9/hoogle work while other snapshots may not, eg: https://beta.stackage.org/lts-1.8/hoogle.

I suspect this has more to do with when the hoogle feature came online, but @snoyberg or @chrisdone would be able to confirm.

**Comment by gitfoxi** [comment]
The form at https://beta.stackage.org just failed for me on the first try but then succeeded on several subsequent tries.

**Comment by snoyberg** [comment]
It's theoretically possible, though highly unlikely, for a random request to fail because the request for the Hoogle database to Amazon S3 to fail. But that's a very rare occurrence. I've been unable to trigger a failing request for the recent snapshots, and old snapshots (more than 5 snapshots old) simply don't have the Hoogle databases generated, so the error message is accurate.

**Comment by ketzacoatl** [comment]
I have updated the ECS task definition to be more specific in running the latest docker image available (and we can use image tags in the future, and test the use of image digests which came out in v1.6.x), and I have not been able to get an error out of hoogle with a little bit of poking. Can anyone else confirm this as an error for current LTS?

Note that, as @snoyberg pointed out, older snapshots don't have the dbs generated.

**Comment by gitfoxi** [comment]
I was just checking it out because of the recent Reddit post. So whatever version was running that day.

**Comment by ketzacoatl** [comment]
Sorry for the confusion @gitfoxi. In simpler terms, I believe the issue is resolved, can you confirm if you still see any errors?

**Comment by gitfoxi** [comment]
I was able to do many searches with no problems.

**Comment by ketzacoatl** [comment]
Great! Thank you for the report and your time testing @gitfoxi.
